api/src/main/java/org/hawkular/inventory/api/Relationships.java

```diff
@@ -54,6 +54,32 @@ private Relationships() {
         owns
     }
 
+
+    /**
+     * The list of possible relationship (aka edges) direction. Relationships are not bidirectional.
+     */
+    public enum Direction {
+        /**
```

Comment:
Actually, I don't like the discrepancy between the "source/target/both" and "incoming/outgoing/both" nomenclature, but I wasn't able to come up with a better meaningful names for the `.relationship(<<value>>)` "sourced" and "targeted" sounds quite confusing to me.

wdyt? What about using the "sourc(ed)/target(ed)/both" when speaking about entities and using the Related filter and "incoming/outgoing/both" when talking about the relationships itself?